### DISCUSSION TITLE
申请退款，完全无法使用

### DISCUSSION THREAD

**Discussion by terryliu1994** [getlantern/lantern-forum-cn#751] [General]
macos windows 都无法使用，一直连接不上， windows的打开就闪退
邮箱：694539215@qq.com
![_FI828_{%GXQ71KD7G~3BNQ](https://github.com/user-attachments/assets/96ed7a0e-1285-436e-bceb-b819eb716c04)

**Comment by MaxMKMax** [comment]
@terryliu1994 您好. 请问您有下载最新版本吗? 无法使用可能是版本的问题. 如果您真的需要退款, 我就帮您提交问题.

**Reply by terryliu1994** [reply]
是最新版，无法使用，申请退款

**Reply by MaxMKMax** [reply]
@terryliu1994 好的. 明白.

**Comment by MaxMKMax** [comment]
@terryliu1994 谢谢您的反馈。蓝灯已经在正视这个问题了。 我们正在修复问题和进行优化。请您耐心等待。非常抱歉影响了您的体验感。谢谢您的等待与支持。

您可以帮助我们在应用程序中提交报告您的问题吗？您可以访问Lantern应用程序并提交log/ 报告。

go to Support (转到支持)
select Report an Issue (选择报告问题)
choose issue type (选择问题类型)
include additional details (包括额外的详细信息)
hit Send Report (点击发送报告)

当您在app提交问题报告后您会收到邮件. 邮件标题上有 8 位数号码. 请回来这里留言您的log/ticket号码. 因为我需要拿您的反馈报告号码来和科技团队沟通您遇到的问题. 不然我无法提交您遇到的问题给科技团队. 不好意思, 需要您的帮忙. 谢谢!

有了这些资料，我们科技团队便可以尽快解决问题。您可以在留言区给我知道。谢谢。

**Reply by xiaolin6456** [reply]
应用程序提交不上报告，没有反应的

**Reply by MaxMKMax** [reply]
@xiaolin6456 
您好. 请问您用的是哪个设备提交报告? 还有那个版本的蓝灯?
